iree/compiler/Dialect/HAL/Target/LLVM/LLVMIRPasses.cpp

```diff
@@ -87,6 +88,28 @@ LogicalResult runLLVMIRPasses(const LLVMTargetOptions &options,
   passBuilder.registerLoopAnalyses(loopAnalysisManager);
   passBuilder.crossRegisterProxies(loopAnalysisManager, functionAnalysisManager,
                                    cGSCCAnalysisManager, moduleAnalysisManager);
+
+  if (options.sanitizerKind == SanitizerKind::kAddress) {
+    bool compileKernel = false;
+    bool recover = false;
+    bool useAfterScope = true;
+    bool moduleUseAfterScope = false;
+    bool useOdrIndicator = false;
```

Comment:
They are only used in the lambda. Let's move them into the lambda so we don't have to capture values.